src/externalServices/storeTrigger/requestCall.ts

```diff
@@ -30,4 +31,21 @@ export class StoreTriggerCall {
     });
     return response.data;
   }
+
+  public async deleteModel(request: DeleteRequest): Promise<StoreTriggerResponse> {
+    this.logger.debug({
```

Comment:
Keep the convention from other functions
```suggestion
  public async deleteModel(request: DeleteRequest): Promise<StoreTriggerResponse> {
```

Reply:
?

Reply:
this is the same as the createModel function

Reply:
The request change is on this line:
  public async deleteModel(request: DeleteRequest): Promise<StoreTriggerResponse> {

Reply:
i havent understood.

Reply:
did you meant  public async deleteModel(request: DeleteRequest): Promise<StoreTriggerResponse> {
    this.logger.info({    ?